src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -776,14 +837,17 @@ def restore_global_state(self, saved_state: dict) -> None:
 class CoefficientSimulation(Simulation):
     """Start and control a simulation with coefficient trading."""
 
+    area: "CoefficientArea"
+    _results: CoefficientSimulationResultsManager
+
     def _init(self, redis_job_id: str) -> None:
         # has to be called before load_setup_module():
         global_objects.profiles_handler.activate()
 
         self.area = self._setup.load_setup_module()
 
         self._results.init_results(redis_job_id, self.area, self._setup)
-        self._results.update_and_send_results(
+        self._results.update_send_coefficient_results(
```

Comment:
The problem is that they do not have exactly the same interface. `update_send_coefficient_results` also requires the scm_manager object in order to calculate the results. Due to the fact that the 2 methods have different signatures / number of arguments, method overriding would not be a possibility (it would be, but pylint complains about overriding a method with different number of arguments). So, and in order to avoid using *args and **kwargs as arguments (which would make the method overriding pylint-compatible, but it is much worse IMO) I decided to use another method name, and deal with code duplication. Hope the above is clear let me know if it does not make sense somewhere.

Reply:
🤦  Sorry, I over-read passing the `scm_manager`. Ok, understood. Thanks!